app/(main)/o/[slug]/settings/users/user-settings.tsx

```diff
@@ -161,7 +162,11 @@ export default function UserSettings({
 
   async function onSubmit(values: z.infer<typeof formSchema>) {
     const currentUsage = Number(totalUsers) + Number(totalInvites) + values.emails.length;
-    const needsMoreSeats = effectiveSeats !== undefined && currentPlan !== "developer" && currentUsage > effectiveSeats;
+    const needsMoreSeats =
```

Comment:
I think I saw this below too, if its common enough consider separating it out into a function.